include/net/aws_jobs.h

```diff
@@ -329,6 +329,22 @@ int aws_jobs_update_job_execution(struct mqtt_client *const client,
 			     int expected_version,
 			     const u8_t *client_token);
 
+/**
+ * @brief AWS Jobs get job execution.
+ *
+ * This implements fetching a job from the queue in AWS.
+ *
+ * @param[in] client           Initialized and connected MQTT Client instance.
```

Comment:
Excess indentation of the description, also missing alignment in later lines.

Reply:
Think this is fixed now